content/vault/v1.21.x (rc)/content/docs/troubleshoot/tune-lease-ttl.mdx

```diff
@@ -185,6 +189,28 @@ The following command lists all available parameters that you can tune.
 $ vault secrets tune -h
 ```
 
+</Tab>
+<Tab heading="GUI" group="gui">
+
+> **Note:** Vault 1.21 supports configuration updates (tune) for mounted **keymgmt** secrets engine in the GUI. Support for other engines may be added in future releases.
+
+To tune a keymgmt mount:
+
+1. Navigate to **Secrets Engines** in the Vault GUI.
```

Comment:
If these steps are the same in both places, we should move them to a partial as well so we're always presenting the steps in the same way

Reply:
thank you for calling this out! I've added a partial for the whole GUI section that can be included in both files. thank you again!!